bin/acpp

```diff
@@ -2010,6 +2010,12 @@ if __name__ == '__main__':
     print("acpp requires python 3.")
     sys.exit(-1)
   
+  filename = os.path.basename(os.path.realpath(__file__))
+  if os.path.basename(os.path.realpath(__file__)) == "syclcc":
+    print("acpp warning: syclcc is deprecated; please use acpp instead.")
+  if os.path.basename(os.path.realpath(__file__)) == "syclcc-clang":
```

Comment:
Looks like you left this change half-way :)

```suggestion
  if filename == "syclcc":
    print("acpp warning: syclcc is deprecated; please use acpp instead.")
  if filename == "syclcc-clang":
```

Reply:
:rofl: :cry: In my defense, I tested it and it worked correctly :P